apps/src/StudioApp.js

```diff
@@ -1968,22 +1968,12 @@ StudioApp.prototype.handleEditCode_ = function (config) {
     }
   }
 
-  // Remove maker API blocks from palette and autocomplete, unless maker APIs are enabled.
-  // We didn't have access to project.useMakerAPIs() when dropletConfig
-  // was initialized, so include it initially, and conditionally remove it here.
+  // Remove maker API blocks from palette, unless maker APIs are enabled.
```

Comment:
No longer true - we now know whether maker APIs are to be available early enough to do this in `Applab.init`.